### ISSUE TITLE
Default selected method not supported (dd's)

### ISSUE THREAD

**Issue by sjunges** [moves-rwth/storm#264]
When using storm with dd and default settings, I get a warning that the selected method is not supported, which is confusing.

```
Storm 1.6.5 (dev)

Command line arguments: --prism examples/grid.prism -const N=800 --prop 'Tmin=? [F x=3 & y > 4]' -e dd

Model checking property "1": T[exp]min=? [F ((x = 3) & (y > 4))] ...
 WARN (SymbolicMinMaxLinearEquationSolver.cpp:64): Selected method is not supported for this solver, switching to value iteration.
```

**Comment by tquatmann** [comment]
I see where this is coming from: the default solver is `topological` which is not available for `dd`.

One option would be to change the command line interface so that `topological` is no longer seen as a *type* of solver, but an option for solving that can be enabled/disabled (independent of the solver type). IMO this would be  a bit cleaner and also solves this issue.

**Comment by sjunges** [comment]
Agree! So the option --topological should have an auto, disabled and enabled option, with auto being default and auto translating to enabled on sparse and disabled on dd?
